RELEASENOTES.md

```diff
@@ -1,2 +1,2 @@
+- Renamed the parameter name from migrationId to archiveId for GitHub REST Migrations APIs
```

Comment:
If we merge this change, I think we can leave out the release notes as it's a refactor that doesn't affect the customer.

Reply:
@timrogers thanks! I've remove the release notes!